File: python/8 kyu/131.py
# https://www.codewars.com/kata/55d24f55d7dd296eb9000030/train/python

# Passed

def summation(num):
    result = 0
    for n in range(1, num + 1):
        result += n
    
    return result

output = summation(8)
print(output)

File: nodejs/7 kyu/131.js
// https://www.codewars.com/kata/58bccdf56f25ff6b6d00002f/train/javascript

// Passed

function rounding(n, m) {
  let bestFactor = Math.round(n / m);
  return ((2 * bestFactor - 1) * m) / 2 === n ? n : bestFactor * m;
}

const output = rounding(20, 3);
console.log(output);